drivers/azure/azureutil/tenantid.go

```diff
@@ -41,3 +84,7 @@ func findTenantID(env azure.Environment, subscriptionID string) (string, error)
 	}
 	return m[1], nil
 }
+
+func saveTenantID(path string, tenantID string) error {
+	return ioutil.WriteFile(path, []byte(tenantID), 0600)
```

Comment:
Can we have multiple go routines writing at the same time the same file? Can this be problematic?